src/Utils/Products-filteration.js

```diff
@@ -0,0 +1,42 @@
+export const getSortedProducts = (originalData, sortBy) => {
+    const productList = [...originalData];
+    if (sortBy && sortBy === "HIGH_TO_LOW") {
+        return productList.sort((a, b) => b.finalPrice - a.finalPrice);
+    }
+    if (sortBy && sortBy === "LOW_TO_HIGH") {
+        return productList.sort((a, b) => a.finalPrice - b.finalPrice);
```

Comment:
sort is an impure function. You can do [...productList].sort((a, b) => b.finalPrice - a.finalPrice).This will return a new array.

Reply:
Ok Thanks @shrushti2000  I did this thing